facter/facter.go

```diff
@@ -140,11 +141,18 @@ func (f *Facter) GetMetricTypes(_ plugin.PluginConfigType) ([]plugin.PluginMetri
 // Collect collects metrics from external binary a returns them in form
 // acceptable by Snap, only returns collects that were asked for and return nothing when asked for none
 // the order of requested and received metrics isn't guaranted
-func (f *Facter) CollectMetrics(metricTypes []plugin.PluginMetricType) ([]plugin.PluginMetricType, error) {
+func (f *Facter) CollectMetrics(metricTypes []plugin.MetricType) ([]plugin.MetricType, error) {
 
 	// parse and check requested names of metrics
 	names := []string{}
+	tags := make(map[string]string)
 	for _, metricType := range metricTypes {
+		tags = metricType.Tags()
+		if tags == nil {
+			tags = map[string]string{}
+		}
+		host, _ := os.Hostname()
+		tags["hostname"] = host
```

Comment:
Just want to make sure that you like to add the hostname for each metric type, not the metric value.